### PR TITLE
fix pre-commit hooks in the CI

### PR DESCRIPTION
**Pull request by v1v**
## Highlights
- gometalinter is deprecated
- go-build/go-unit-test should be done within the pipeline itself rather than within the pre-commit.
- go-vet causes some issues when loading some packages. Let's remove it from now

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -61,6 +60,7 @@ pipeline {
                   unstash 'source'
                   dir(BASE_DIR){
                     sh script: """.ci/scripts/install-dependencies.sh "${GO_VERSION}" """, label: 'Install dependencies'
+                    sh 'ls -ltrah && pwd && env | sort'
```
Comment (mdelapenya) [inline .ci/Jenkinsfile]: Do we want `env` to be executed on CI?

File: cli/config/config.go [hunk 2]
```diff
@@ -10,6 +10,10 @@ import (
 	"reflect"
 	"strings"
 
+
```
Comment (mdelapenya) [inline cli/config/config.go]: I guess this is for CI to complain 😄

File: .ci/Jenkinsfile (hunk 1 above)
Comment (v1v) [reply .ci/Jenkinsfile]: debugging yet as it works locally but not in the CI

```
14:39:51  hookid: golangci-lint
14:39:51  
14:39:51  level=error msg="Running error: context loading failed: no go files to analyze"
```

File: cli/config/config.go (hunk 2 above)
Comment (v1v) [reply cli/config/config.go]: exactly

File: .ci/Jenkinsfile (hunk 1 above)
Comment (v1v) [inline .ci/Jenkinsfile]:
```suggestion
```

**Pull request merged**